pkg/apis/internal.admin.acorn.io/v1/resources.go

```diff
@@ -7,65 +7,97 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+type (
+	Flag            string
+	Count           string
+	PersistentCount string
+	Quantity        string
```

Comment:
I recommend adding a comment explaining the meaning of each of these.

Reply:
Done!